src/openforms/forms/admin/mixins.py

```diff
@@ -20,7 +22,7 @@ class FormioConfigMixin:
     def render_change_form(
         self, request, context, add=False, change=False, form_url="", obj=None
     ):
-        config = GlobalConfiguration.get_solo()
+        config = cast(GlobalConfiguration, GlobalConfiguration.get_solo())
```

Comment:
Same here. If we don't want to trip up production with an `AssertionError`, we can run with -O and wait for the bug report with weird behaviour. Or we can catch it and log it to Sentry and then let it go in with its unexpected behaviour.

I'm not saying the cast is wrong here. But code has a tendency to get copied. I've caught faulty casts before.
I'd rather take the performance hit of an assert, than have a cheap cast to make the type checker give me false confidence.

I do love the goal btw... just not the means.

Reply:
if it trips up production with `AssertionError`, we have bigger fish to fry!